apps/react-18-tests-v9/cypress.config.ts

```diff
@@ -1,3 +1,27 @@
+import * as path from 'path';
 import { baseConfig } from '@fluentui/scripts-cypress';
 
-export default baseConfig;
+const specs = [
+  path.resolve('../../apps/react-18-tests-v9/src/**/*.cy.{tsx,ts}'),
+  path.resolve('../../packages/react-components/**/library/src/**/*.cy.{tsx,ts}'),
+
+  // Exclude files that are not compatible with React 18 yet
+  '!' + path.resolve('../../packages/react-components/react-overflow/library/src/**/*.cy.{tsx,ts}'),
+  '!' + path.resolve('../../packages/react-components/react-tag-picker/library/src/**/*.cy.{tsx,ts}'),
+  '!' + path.resolve('../../packages/react-components/react-toolbar/library/src/**/*.cy.{tsx,ts}'),
+];
+
+const config = { ...baseConfig };
+
+config.component.specPattern = specs;
+config.component.devServer.webpackConfig.resolve ??= {};
+config.component.devServer.webpackConfig.resolve.alias = {
+  ...config.component.devServer.webpackConfig.resolve.alias,
+  '@cypress/react': path.resolve(__dirname, './node_modules/@cypress/react'),
+  '@types/react': path.resolve(__dirname, './node_modules/@types/react'),
```

Comment:
curious if  cypress is even running tsc ?

Reply:
It is, because we have tests in the package itself. Bumping Cypress alone was making them raise warnings when running inside vscode. To keep consistency, I added the types here too.

This will also be helpful in the future to add type tests too.